hack/helm/release/cmd/root.go

```diff
@@ -0,0 +1,172 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/elastic/cloud-on-k8s/hack/helm/release/internal/helm"
+)
+
+const (
+	// viper flags
+	bucketFlag          = "bucket"
```

Comment:
```suggestion
	bucketFlag          = "bucket"
```

Reply:
What's the intention here?  I see no changes suggested.

Reply:
Oops, the constant can be deleted, it is no longer used.